<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Aerolinea XYZ - Inicio de Sesión </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 60, 90); color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/logo.png</pixmap> <!-- Inserta el logo de la empresa aquí -->
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Aerolinea XYZ - Inicio de Sesión</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt "Segoe UI"; color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_usuario">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt "Segoe UI";</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_usuario">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese su usuario</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_contrasena">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt "Segoe UI";</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_contrasena">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Ingrese su contraseña</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255); color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_aceptar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 123, 255); font: bold 12pt "Segoe UI"; color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cancelar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 69, 0); font: bold 12pt "Segoe UI"; color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0); font: italic 10pt "Segoe UI";</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
